src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -119,7 +120,7 @@ public void loadAllergiesPage() {
 
 	@Then("the system loads Allergies board page")
 	public void systemLoadsAllergiesPage() {
-		assertTrue(textExists("Allergies"));
```

Comment:
Why changing this assertion

Reply:
> 

@sherrif10 we just want to do away with relying on displayed texts, such that the tests can still pass even in another locale where English is not the language used.